utils/index.js

```diff
@@ -7,12 +7,16 @@ import axios from 'axios';
 import { promises as fs } from 'fs';
 import { SocksProxyAgent } from 'socks-proxy-agent';
 
-import { PROXY_TEST_TIMEOUT, PROXY_TESTING_FILE } from './const.js';
-import { getProxyForKey, getRandomKey } from './mysql-queries.js';
+import { PROXY_TEST_TIMEOUT, PROXY_TESTING_FILE } from '../constants/index.js';
+import { getProxyForKey, getRandomKey } from '../utils/mysql-queries/index.js';
 import { InfoMessage } from './custom-errors.js';
 
+export { InfoMessage } from './custom-errors.js';
+
+// TODO: split into files?
 export const delay = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
 
+// TODO: move somewhere?
```

Comment:
This may not be needed. I added it for connection to elastic w/ unknown cert. But we are probably using different setup now?